sentry/src/main/java/io/sentry/SentryWrapper.java

```diff
@@ -27,16 +27,18 @@ public final class SentryWrapper {
    * @return the wrapped {@link Callable}
```

Comment:
Comment refers to `Hub`, should refer to `Scopes`

Reply:
Changed in https://github.com/getsentry/sentry-java/pull/3366